test/fetch_helpers.jl

```diff
@@ -0,0 +1,19 @@
+using Test
+using DataDeps: fetch_default, fetch_base, fetch_http
+
+ENV["DATADEPS_ALWAYS_ACCEPT"] = true
+
+@testset "easy https url" begin
+    url = "https://www.angio.net/pi/digits/10000.txt"
+    # This is easy because the filename is in the URL
+    # So it works with both `fetch_base` and `fetch_http`
+    # HTTP.jl has tests for much more difficult cases, and fetch_http supports those
+    @testset "$fetch" for fetch in (fetch_default, fetch_base, fetch_http)
```

Comment:
Pretty minor, but I might be inclined to rename `fetch` to `f` or `func` to make it clear that we aren't trying to call the base `fetch` method in this loop.

Reply:
maybe `fetch_func`